doc/api/repl.md

```diff
@@ -411,6 +417,8 @@ added: v0.1.91
     `SIGINT` is received, i.e. `Ctrl+C` is pressed. This cannot be used together
     with a custom `eval` function. Defaults to `false`.
 
+If `options` is a string, then it specifies the input prompt.
+
```

Comment:
Don't know whether this is true. If so it would be a major change in documentation, since also the type is just displaying <Object>